app/views/shared/_national_statistics_logo.html+new_navigation.erb

```diff
@@ -0,0 +1,3 @@
+<div class="national-statistics-logo">
```

Comment:
Why does this logo need a `new_navigation` file? Is it different in the A and B versions?